From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Fri, 24 Jun 2016 16:51:41 +0200
Subject: [PATCH] Cleanup: Use dget instead of wget+dpkg-source.

We already install devscripts, which provides dget.

See [1] for using `dget -x -u` to accelerate/simplify code dev.

FIXME: Add back `-u` when we add Ubuntu signing keys to builder
docker keyring.

[1] https://debian-administration.org/article/
    504/Downloading_Debian_source_packages_easily

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 ovs_build/build-ovs-dpdk.sh     | 10 ++--------
 ovs_build/build-ovs-nsh-dpdk.sh | 10 ++--------
 2 files changed, 4 insertions(+), 16 deletions(-)

diff --git a/ovs_build/build-ovs-dpdk.sh b/ovs_build/build-ovs-dpdk.sh
index 83f4278..8119fb2 100755
--- a/ovs_build/build-ovs-dpdk.sh
+++ b/ovs_build/build-ovs-dpdk.sh
@@ -17,10 +17,7 @@ rm -rf ${BUILD_HOME}; mkdir -p ${BUILD_HOME}
 cp `dirname $0`/rules ${BUILD_HOME}
 
 cd ${BUILD_HOME}
-wget -c https://launchpad.net/ubuntu/+archive/primary/+files/dpdk_2.2.0-0ubuntu8.dsc
-wget -c https://launchpad.net/ubuntu/+archive/primary/+files/dpdk_2.2.0.orig.tar.gz
-wget -c https://launchpad.net/ubuntu/+archive/primary/+files/dpdk_2.2.0-0ubuntu8.debian.tar.xz
-dpkg-source -x dpdk_2.2.0-0ubuntu8.dsc
+dget -x -u https://launchpad.net/ubuntu/+archive/primary/+files/dpdk_2.2.0-0ubuntu8.dsc
 wget -c http://fast.dpdk.org/rel/dpdk-16.04.tar.xz
 
 # copy from debian/control
@@ -53,10 +50,7 @@ cd ${BUILD_HOME}; sudo dpkg -i *.deb
 apt-get download libxenstore3.0
 
 cd ${BUILD_HOME}
-wget -c https://launchpad.net/ubuntu/+archive/primary/+files/openvswitch-dpdk_2.4.0.orig.tar.gz
-wget -c https://launchpad.net/ubuntu/+archive/primary/+files/openvswitch-dpdk_2.4.0-0ubuntu1.dsc
-wget -c https://launchpad.net/ubuntu/+archive/primary/+files/openvswitch-dpdk_2.4.0-0ubuntu1.debian.tar.xz
-dpkg-source -x openvswitch-dpdk_2.4.0-0ubuntu1.dsc
+dget -x -u https://launchpad.net/ubuntu/+archive/primary/+files/openvswitch-dpdk_2.4.0-0ubuntu1.dsc
 
 # copy from debian/control
 sudo apt-get install -y autoconf \
diff --git a/ovs_build/build-ovs-nsh-dpdk.sh b/ovs_build/build-ovs-nsh-dpdk.sh
index 5ac2909..0c09e46 100755
--- a/ovs_build/build-ovs-nsh-dpdk.sh
+++ b/ovs_build/build-ovs-nsh-dpdk.sh
@@ -14,10 +14,7 @@ sudo apt-get -y install devscripts dpkg-dev git wget
 rm -rf ${BUILD_HOME}; mkdir -p ${BUILD_HOME}
 
 cd ${BUILD_HOME}
-wget -c https://launchpad.net/ubuntu/+archive/primary/+files/dpdk_2.2.0-0ubuntu8.dsc
-wget -c https://launchpad.net/ubuntu/+archive/primary/+files/dpdk_2.2.0.orig.tar.gz
-wget -c https://launchpad.net/ubuntu/+archive/primary/+files/dpdk_2.2.0-0ubuntu8.debian.tar.xz
-dpkg-source -x dpdk_2.2.0-0ubuntu8.dsc
+dget -x -u https://launchpad.net/ubuntu/+archive/primary/+files/dpdk_2.2.0-0ubuntu8.dsc
 
 # copy from debian/control
 sudo apt-get install -y debhelper \
@@ -46,10 +43,7 @@ cd ${BUILD_HOME}; sudo dpkg -i *.deb
 apt-get download libxenstore3.0
 
 cd ${BUILD_HOME}
-wget -c https://launchpad.net/ubuntu/+archive/primary/+files/openvswitch-dpdk_2.4.0.orig.tar.gz
-wget -c https://launchpad.net/ubuntu/+archive/primary/+files/openvswitch-dpdk_2.4.0-0ubuntu1.dsc
-wget -c https://launchpad.net/ubuntu/+archive/primary/+files/openvswitch-dpdk_2.4.0-0ubuntu1.debian.tar.xz
-dpkg-source -x openvswitch-dpdk_2.4.0-0ubuntu1.dsc
+dget -x -u https://launchpad.net/ubuntu/+archive/primary/+files/openvswitch-dpdk_2.4.0-0ubuntu1.dsc
 
 # copy from debian/control
 sudo apt-get install -y autoconf \
